### PR TITLE
[DOC]update outofdate mortgage notebooks and update docs for xgboost161 jar[skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
update new mortgage-ETL notebooks for GCP/DB/EMR and update docs for xgboost161 jar
this pr merged #5778 and #5990

Signed-off-by: liyuan <yuali@nvidia.com>

### CODE REVIEW

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link. Download the sample 
-[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) and upload it in Jupyter.
+[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL.ipynb) and upload it in Jupyter.
 
 To get the input data of the sample notebook, we need to download the full 17 years' [Mortgage
```
Comment (SurajAralihalli) [inline docs/get-started/getting-started-gcp.md:147]: The getting started guides for csp still point to the datasets hosted on rapids.ai 
We can use these [instructions](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.08/docs/get-started/xgboost-examples/dataset/mortgage.md) to download the dataset.

File: docs/get-started/getting-started-databricks.md [hunk 2]
```diff
@@ -169,7 +169,7 @@ cluster.
 8. Once the cluster comes back up, it is now enabled for GPU-accelerated Spark.
 
 ## Import the GPU Mortgage Example Notebook
-Import the example [notebook](../demo/gpu-mortgage_accelerated.ipynb) from the repo into your
+Import the example [notebook](../demo/Databricks/Mortgage-ETL-db.ipynb) from the repo into your
 workspace, then open the notebook.  Modify the first cell to point to your workspace, and download a
 larger dataset if needed. You can find the links to the datasets at
 [docs.rapids.ai](https://docs.rapids.ai/datasets/mortgage-data).
```
Comment (SurajAralihalli) [inline docs/get-started/getting-started-databricks.md:175]: We can use these [instructions](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.08/docs/get-started/xgboost-examples/dataset/mortgage.md) to download the dataset.

File: docs/get-started/getting-started-gcp.md (hunk 1 above)
Comment (nvliyuan) [reply docs/get-started/getting-started-gcp.md]: updated

File: docs/get-started/getting-started-databricks.md (hunk 2 above)
Comment (nvliyuan) [reply docs/get-started/getting-started-databricks.md]: updated

**Review by viadea — DISMISSED** [review]

**Comment by nvliyuan** [comment]
@viadea @SurajAralihalli just address the comment and add convert raw CSV files to parquet in notebooks to improve performance, can you please help review again?

**Review by SurajAralihalli — APPROVED** [review]

**Review by viadea — APPROVED** [review]

**Comment by tgravescs** [comment]
build

**Pull request merged**